Gemfile.lock

```diff
@@ -361,6 +390,7 @@ DEPENDENCIES
   rails!
   rake (>= 10.3)
   redcarpet (~> 3.2.3)
+  redis (~> 3.0)
```

Comment:
I think the redis requirement can just kick in when we start the server
with ./bin/cable or whatever. And then the redis gem file can be listed by
default in the default Gemfile for new apps. And if someone tries to start
bin/cable without that, we'll complain about why.

On Mon, Dec 14, 2015 at 4:57 PM, Sean Griffin <notifications@github.com>
wrote:

> In Gemfile.lock
> <https://github.com/rails/rails/pull/22585#discussion_r47514517>:
>
> > @@ -361,6 +390,7 @@ DEPENDENCIES
> >    rails!
> >    rake (>= 10.3)
> >    redcarpet (~> 3.2.3)
> > +  redis (~> 3.0)
>
> I'm not sure that'll work here, since we conditionally require the adapter
> based on the configuration, but Action Cable will always end up being
> required. I'm concerned that this will cause a large number of existing
> apps to be unable to upgrade by default, if Redis isn't already installed
> on the machine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/22585/files#r47514517>.
>